recipes/nlopt/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "2.4.2" %}
+
+package:
+  name: nlopt
+  version: {{ version }}
+
+source:
+  fn: nlopt-{{ version }}.tar.gz
+  url: http://ab-initio.mit.edu/nlopt/nlopt-{{ version }}.tar.gz
+  md5: d0b8f139a4acf29b76dbae69ade8ac54
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python {{PY_VER}}*  # [not win]
+    - numpy  # [not win]
+
+  run:
+    - python {{PY_VER}}*  # [not win]
```

Comment:
@ocefpaf I am sure that I needed the {{PY_VER}}* at the time and that it broke openturns depencies because of numpy. Did you read what the explanation I wrote in pr 664 ?